mix.exs

```diff
@@ -7,7 +7,7 @@ defmodule BlockScout.Mixfile do
     [
       # app: :block_scout,
       # aliases: aliases(config_env()),
-      version: "5.2.1",
+      version: "3.1.0",
```

Comment:
Downgrading?

Reply:
I think 5.2.1 is meant to be the Blockscout version we are using (of the original project), not the eon-explorer version

Reply:
The previous number 5.2.1 as @voyenavion has mentioned, is the original blockscout version from where we have forked. 
Our version numbers for some reason have started in 1.0.0 instead of continuing the original repo versioning.
I have set it to 3.1.0 because that is going to be the next release we are going to create. 
We can change this though, and release version 6.0.0 for instance, so that we continue the original versioning.
I'm not clear here myself, but in order to do our checks consistent with our tags right now the correct number here is 3.1.0